lib/commands/run.js

```diff
@@ -59,6 +60,8 @@ function run(scriptPath, options) {
           return callback(new Error(msg2), null);
         }
 
+        script.config = _.defaultsDeep(config, script.config);
```

Comment:
@xicombd this line here is what I meant, it allows the user to implicitly override settings in the actual script file with values from .minigunrc